src/Raven.Server/Documents/Handlers/Admin/AdminConfigurationHandler.cs

```diff
@@ -28,9 +28,10 @@ public class AdminConfigurationHandler : DatabaseRequestHandler
                     var result = await ServerStore.WriteDatabaseRecordAsync(Database.Name, record, index);
                     await Database.RachisLogIndexNotifications.WaitForIndexNotification(result.Etag);
                 }
-
+                
+                NoContentStatus();
```

Comment:
To remove: 
HttpContext.Response.Headers.Remove("Content-Type");
since having empty response with cotnent type set to applicaiton/json creates exception in jquery.